driver/__init__.py

```diff
@@ -1,2 +1,2 @@
 from .webdriver import WebDriver
```

Comment:
Rename `driver` directory to `library` or `client`. This is not a web driver but a client library for WebDriver protocol.